packages/git/src/browser/git-repository-tracker.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject, postConstruct } from "inversify";
+import { Git, Repository, WorkingDirectoryStatus } from '../common';
+import { Event, Emitter, DisposableCollection } from "@theia/core";
+import { GitRepositoryProvider } from './git-repository-provider';
+import { GitWatcher, GitStatusChangeEvent } from "../common/git-watcher";
+
```

Comment:
I really like this, but we have server watcher, client watcher, watcher protocol, tracker... perhaps it worth a sentence as a JS doc what it does. So newcomers will know, **this** is what I have to use...

Reply:
Good point.

Reply:
@kittaakos I've updated the repository tracker and added some documentation.